packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -95,6 +98,17 @@ enum ImportTokenModalStatus {
   CLOSED
 }
 
+const networkConnectionWarningToast = () =>
+  warningToast(
```

Comment:
Wouldn't it make more sense not to `autoClose` messages that prompt users to contact support?

Reply:
when they hover the toast, the counter pauses
i think that's good enough?

Reply:
hm I think it should be good enough in this case, but maybe for important errors it would be nice to just keep it open.

Also when a user goes afk and they will just come back unaware of an error